axelor-helpdesk/src/main/java/com/axelor/apps/helpdesk/web/TicketController.java

```diff
@@ -249,4 +252,72 @@ public void timerStateOff(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void setStartStatus(ActionRequest request, ActionResponse response) {
+    try {
+      Ticket ticket = request.getContext().asType(Ticket.class);
+
+      Beans.get(TicketWorkflowService.class).startTicket(ticket);
+      response.setValue("ticketStatus", ticket.getTicketStatus());
+    } catch (Exception e) {
+      TraceBackService.trace(response, e);
+    }
```

Comment:
Starting AOS v7, we don't need to use the try catch anymore in Controllers, you can remove it to save some lines.